openctidata/config.yml.sample

```diff
@@ -0,0 +1,11 @@
+rabbitmq:
+  hostname: 'localhost'
+  port: 5672
+  username: 'guest'
+  password: 'guest'
+
+openctidata:
+  sectors_file_url: 'https://raw.githubusercontent.com/OpenCTI-Platform/datasets/master/data/sectors.json'
```

Comment:
Still confuse why we dont have a connector capable of fetching all the repository.

Reply:
On disconnected network, a Git repository is not always available. I prefer provide direct links that user are able to customize if they need to host the file somewhere else.